src/components/layouts/pages/inspect/tree/Tree.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState, Fragment } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { getRuleSummaries } from '../../../../../store/job/result/selectors';
+import { getJobProfile } from '../../../../../store/job/selectors';
+import List from '@material-ui/core/List';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Collapse from '@material-ui/core/Collapse';
+import Tooltip from '@material-ui/core/Tooltip';
+import Chip from '@material-ui/core/Chip';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+
+import './Tree.scss';
+import errorMap from '../validationErrorMessages_en.json';
+
+const LIST_HEADER = 'Errors overview';
+const UNSELECTED = -1;
+
+function Tree({ ruleSummaries, profile }) {
+    const [openedRule, setOpenedRule] = useState(UNSELECTED);
+    const [selectedCheck, setSelectedCheck] = useState(UNSELECTED);
+    const onRuleClick = index => {
+        if (openedRule === index) {
+            return setOpenedRule(UNSELECTED);
+        }
+
+        return setOpenedRule(index);
+    };
+    const onCheckClick = context => setSelectedCheck(context);
+
+    return (
+        <section className="summary-tree">
+            <List
+                aria-labelledby="summary-tree-subheader"
+                subheader={
+                    <ListSubheader component="div" id="summary-tree-subheader" disableSticky>
+                        {LIST_HEADER}
+                    </ListSubheader>
+                }
+            >
+                <RuleList
+                    profile={profile}
+                    ruleSummaries={ruleSummaries}
+                    openedRule={openedRule}
+                    selectedCheck={selectedCheck}
+                    onRuleClick={onRuleClick}
+                    onCheckClick={onCheckClick}
+                />
+            </List>
+        </section>
+    );
+}
+
+// TODO: add Warnings
+function RuleList({ ruleSummaries, openedRule, selectedCheck, profile, onRuleClick, onCheckClick }) {
+    return ruleSummaries.map((rule, index) => {
+        const ruleTitle = getRuleTitle({ ...rule, profile });
+        return (
+            <Fragment key={index}>
+                <Tooltip
+                    placement="bottom-start"
+                    enterDelay={500}
+                    enterNextDelay={500}
+                    title={getRuleDescription({ ...rule, profile })}
+                >
+                    <ListItem button onClick={() => onRuleClick(index)} className="rule-item rule-item_error">
+                        {rule.checks.length ? openedRule === index ? <ExpandLess /> : <ChevronRightIcon /> : []}
```

Comment:
Let's use `ExpandMore` icon instead of `ExpandLess` to make component look more like tree. `ExpandLess` was OK when it was in the end, but in the start it is confusing.